web-components/package.json

```
@@ -1,21 +1,37 @@
 {
-  "name": "@lit/lit-starter-ts",
-  "private": true,
+  "name": "openfoodfacts-webcomponents",
+  "author": "Open Food Facts",
+  "license": "LGPL-2.1-only",
```

It's better to be open to future enhancement of the license.

```suggestion
  "license": "LGPL-2.1-or-later",
```